script.filecleaner/default.py

```diff
@@ -0,0 +1,707 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import json
+
+import xbmcvfs
+from utils import *
+
+
+# Addon info
+__addonID__ = "script.filecleaner"
```

Comment:
A note for the next update, please avoid using \__name\__ style labels as these are meant for python internals.